src/components/Requirements/RequirementDescription.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div v-if="nestedRequirements">
+    <div class="nested-requirement-choices">
+      <label class="single-choice" :for="`compound-requirement--id-${requirement.id}--default`">
+        <input
+          :id="`compound-requirement--id-${requirement.id}--default`"
+          type="radio"
+          :checked="choice === ''"
+          @change="() => $emit('update-choice', '')"
+          value="name"
+        />
+        {{ requirement.description }}
+      </label>
+      <label
+        v-for="name in nestedRequirements"
+        :key="name"
+        class="single-choice"
+        :for="`compound-requirement--id-${requirement.id}--choice-${name}`"
+      >
+        <input
+          :id="`compound-requirement--id-${requirement.id}--choice-${name}`"
+          type="radio"
+          :checked="choice === name"
+          @change="() => $emit('update-choice', name)"
+          value="name"
+        />
+        {{ name }}
+      </label>
+    </div>
+    <div class="progress-string">{{ progressString }}</div>
+  </div>
+  <div v-else>
+    {{ requirement.description }}
+    <a :style="{ color: `#${color}` }" :href="requirement.source" target="_blank">
+      <strong>Learn More</strong></a
+    >
+  </div>
+</template>
+
+<script lang="ts">
+import { PropType, defineComponent } from 'vue';
+
+/** Show compound requirement for toggle or description */
```

Comment:
Thanks for this documentation! I think it would be good practice for devs to add a docstring of what the component is/does either above the `export default defineComponent` line or at the top of this file